.github/workflows/neon-preview-branches-for-pull-requests.yml

```diff
@@ -30,6 +30,15 @@ jobs:
     name: Create/Delete Branch for Pull Request Job
     runs-on: ubuntu-latest
     steps:
+      - name: Fetch integration data
+        shell: bash
+        run: |
+          WORKFLOW=$(curl -H "accept: application/json" -H "Authorization: Bearer ${{ secrets.NEON_API_KEY }}" https://console.neon.tech/api/v2/projects/${{ inputs.project_id }}/applications/github | jq -r ".workflows[] | select(.type == \"pr_branches\")")
+          echo $WORKFLOW
+          echo "workflow_parent=$(echo $WORKFLOW | jq .template_values.parent)" >> $GITHUB_ENV
```

Comment:
why don't you choose `$GITHUB_OUTPUT` instead of `$GITHUB_ENV`?

Reply:
Changed to `$GITHUB_OUTPUT`.